poetwp.php

```diff
@@ -30,53 +29,32 @@
 	die;
 }
 
-/**
- * Currently plugin version.
- * Start at version 1.0.0 and use SemVer - https://semver.org
- * Rename this for your plugin and update it as you release new versions.
- */
-define( 'POETWP_VERSION', '1.0.0' );
+// Include the file that defines the custom post type functions.
+require_once plugin_dir_path( __FILE__ ) . 'includes/post-type-functions.php';
```

Comment:
This `plugin_dir_path( __FILE__ )` is called four times - it won't change between each call, I promise - so call it once into a constant (that can also be used elsewhere if needed) and use that in your `require_once` statements.